### PR TITLE
Tests for multi cruncher

### PR DESCRIPTION
**Pull request by reidmcl**
Adds tests for multi_cruncher
Refactors multi_cruncher module design, adds new method with ` get_pol() `
Updates to example jupyter notebook detailing 4 potential user paths (no reform, github preset, dictionary, local file)
Added an example input file for testing purposes

### CODE REVIEW

File: taxcrunch/multi_cruncher.py [hunk 1]
```diff
@@ -84,14 +84,31 @@ def read_input(self):
...
+        mtr_df = pd.DataFrame(data=mtr).transpose()
+        df_res = pd.concat([calcs, mtr_df], axis=1)
+        df_res.columns = self.labels
+        df_res.index = range(self.rows)
+        return df_res
+
+    def get_pol(self, reform_file):
+       
+        # if a reform file is not specified, the default policy is current law
         """
```
Comment (Peter-Metz) [inline taxcrunch/multi_cruncher.py]: Can you move this docstring to the `create_table()` method and create a new docstring for the `get_pol()` method?

File: taxcrunch/tests/test_multi_cruncher.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import pytest
+import json
+import os
+import numpy as np
+import pandas as pd
+import taxcalc as tc
+import taxcrunch.cruncher as cr
+import taxcrunch.multi_cruncher as mcr
+
+def test_create_data():
```
Comment (Peter-Metz) [inline taxcrunch/tests/test_multi_cruncher.py]: To streamline things a bit, you could remove this test and add `assert isinstance(b, mcr.Batch)` to the `test_get_pol_no_reform()` test

**Comment by reidmcl** [comment]
@Peter-Metz Please see changes to docstring for multi_cruncher, additional description for ` get_pol() ` can be added if beneficial

**Comment by Peter-Metz** [comment]
Thanks very much @reidmcl! Merging.

**Pull request merged**